tools/docker/tests/test_flows.sh

```diff
@@ -198,7 +198,7 @@ test_ap_capability_query() {
     check send_CAPI_command gateway "DEV_SEND_1905,DestALid,$mac_agent1,MessageTypeValue,0x8001" $redirect
     sleep 1
     dbg "Confirming ap capability query has been received on agent"
-    check docker exec repeater1 sh -c 'grep -i -q "AP_CAPABILITY_QUERY_MESSAGE" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
```

Comment:
Please also update the controller to log the details of the AP Capability Report it receives, and check that in test_flows.

Reply:
Sorry, but I need a detailed explanation on how to do this

Reply:
@mariomaz Did you get the answer you were looking for?

Reply:
No, I didn't

Reply:
Added handler in controller and check in test_flows.sh